editor/plugins/script_text_editor.cpp

```diff
@@ -690,7 +690,20 @@ void ScriptTextEditor::_update_bookmark_list() {
 	bookmarks_menu->add_separator();
 
 	for (int i = 0; i < bookmark_list.size(); i++) {
+		// Strip edges to avoid rendering unknown ASCII symbol ('�')
 		String line = code_editor->get_text_edit()->get_line(bookmark_list[i]).strip_edges();
+
+		if (line.length() > 1 && line.get(0) == '#') {
+			// Use configured tab size to create a tab replacement using only spaces
+			String tab_replacement = "";
+			for (int j = 0; j < code_editor->get_text_edit()->get_indent_size(); j++) {
+				tab_replacement += " ";
+			}
```

Comment:
That's ok, but IMO for the bookmarks it would have been just as fine to simply replace `"\t"` by one or two spaces.

We don't really need to see all the indentation as wide as it is in the script editor, in fact we even strip it away from indented strings which aren't comments.